src/views/Dashboard.vue

```diff
@@ -210,4 +210,21 @@ export default {
 .v-list__tile {
   padding: 10px 0;
 }
+
+.v-list-resize-left, 
+.v-list-resize-right {
+  padding: 0;
+  margin: 0 12px;
```

Comment:
I think this has been fixed in https://github.com/cylc/cylc-ui/pull/275, just missing @ashnewport to add his name to contributors before being merged.